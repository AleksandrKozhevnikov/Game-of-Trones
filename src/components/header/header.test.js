import React from 'react';
import Header from './header';
import {shallow} from 'enzyme';


describe('Testing <Header/>', () => {
    it('Header is rendered correctly', () => {
        const header = shallow(<Header/>);
        expect(header).toMatchSnapshot();
    })
})