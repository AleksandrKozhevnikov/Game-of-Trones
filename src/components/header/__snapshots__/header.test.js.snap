// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Header/> Header is rendered correctly 1`] = `ShallowWrapper {}`;
